generator/lib/src/writer/writer.dart

```diff
@@ -948,6 +949,8 @@ class Writer {
 
     _write('return await $beanName.removeWhere(exp);');
     _writeln('}');
+    _writeln('return 0;');
```

Comment:
It should return a Future here

Reply:
async functions in dart returns Future by default. I can wrap it in Future.value if it's preferred, however [effective dart](https://www.dartlang.org/guides/language/effective-dart/usage#prefer-asyncawait-over-using-raw-futures) guidlines suggests just returning value.

Reply:
True :) my bad all good then ^^